career/ladder.md

```diff
@@ -23,6 +23,13 @@ Here are some examples of other career ladders:
 ## Dev Values: Impactful, Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled
 
 ### Level 1 - Associate Software Engineer
+As an Associate Software Engineer, you demonstrate an ability to complete work
+on a task-level in one or multiple apps, and not at least an eagerness to learn
```

Comment:
Lists with two items do not need a comma :) 
```suggestion
on a task-level in one or multiple apps and not at least an eagerness to learn
```

Reply:
Thank you!